scripts/circle-ci/run-circle-tests.sh

```diff
@@ -13,15 +13,15 @@ function checkDocsBuild {
 
 CONTAINER_1=(':atlasdb-cassandra-integration-tests:check')
 
-CONTAINER_2=(':atlasdb-ete-tests:check' ':atlasdb-ete-test-utils:check')
+CONTAINER_2=(':atlasdb-ete-tests:check')
 
-CONTAINER_3=(':atlasdb-dbkvs:check' ':atlasdb-perf:postgresBenchmarkTest' ':atlasdb-cassandra:check')
+CONTAINER_3=(':atlasdb-dbkvs:check' ':atlasdb-cassandra:check')
 
-CONTAINER_4=(':atlasdb-cassandra-multinode-tests:check' ':atlasdb-impl-shared:check' ':atlasdb-cassandra-integration-tests:longTest' ':atlasdb-tests-shared:check' ':atlasdb-perf:check')
```

Comment:
we got rid of the `atlasdb-cassandra-integration-tests:longTest`? I don't see it on the other side.

Reply:
It's part of :atlasdb-cassandra-integration-tests:check run on container 1, before it was being excluded.